apps/06SpaceTravelToMars.js

```diff
@@ -9,7 +9,29 @@ var jobTypes = {
   programmer: 'Any Ship!'
 };
 
-// Your code here
+function CrewMember(name, job, specialSkill){
+  this.name = name;
+  this.job = job;
+  this.specialSkill = specialSkill;
+  this.ship = null;
+  this.enterShip = function(crewmember) {
+    this.crew.push(crewmember);
+  }
```

Comment:
It is not. Will add. 

Dully noted.

Ship, I presume, since you are entering a ship.